Aurora Editor.xcodeproj/project.pbxproj

```diff
@@ -3686,7 +3686,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export PATH=\"$PATH:/opt/homebrew/bin\"\nif which swiftlint >/dev/null; then\n  swiftlint --strict\nelse\n  echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
+			shellScript = "export PATH=\"$PATH:/opt/homebrew/bin\"\nif which swiftlint >/dev/null; then\n  swiftlint\nelse\n  echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
```

Comment:
[Review note]

In my Xcode, I couldn't succeed to compile without this modification.
`--strict` make compilation failed when even only sinle wanring happens from SwiftLint, in my understanding.

So I would like you to check if this modification is okay or not.

Reply:
We keep strict enabled, so that we have clean and understandable code.

Reply:
Thank you for your reply. I agree with you, but I guess I can't run AuroraEditor due to this strict linter.

> What issues are you having tho?

This is the error in Xcode though it is unclear for me.

```swift
Command PhaseScriptExecution failed with a nonzero exit code
```

<img width="800" alt="Screenshot 2023-02-12 at 13 56 41" src="https://user-images.githubusercontent.com/44002126/218293799-4de018e0-e9df-4e98-9028-429e0805db56.png">

To debug it, I ran `swiftlint --strict` on my CLI and got error status (3).

<img width="663" alt="Screenshot 2023-02-12 at 13 56 26" src="https://user-images.githubusercontent.com/44002126/218293843-d3bc5dcb-db7d-4477-b192-9449a27ec466.png">

Reply:
If other members can do well, I can discard this change and create new issue for this topic.

Reply:
Do you have the latest version of swiftlint? That may be the issues because in your first screenshot you have an ungodly amount of swiftlint issues🥲

Reply:
@nanashili 
I am so sorry..! I used very old version of SwiftLint...
After updating to the latest version, it works well!
Thank you so much.:pray:

Reply:
https://github.com/AuroraEditor/AuroraEditor/pull/423 This should prevent this things to happen in the future.